facebook/ThreatExchange
# Writebacker framework

Summary
---------

A framework for how writebacks should happen (formerly reactions). See tests for how to use

Test Plan
---------
1. Find an image in the copydays dataset from this website https://lear.inrialpes.fr/~jegou/data.php
2. Go to hma locally and upload file


Added tests
end to end test:
1. upload docker and update all lambdas
2. go to hma in UI and upload a copy days image
3. get hash from match details page and use `bunnylol threatdata <hash>` to get the indicator
4. Look for the indicator and then see that reactions happened
